### PR TITLE
Don't ignore comments in .codeclimate.yml

### PR DESCRIPTION
**Pull request by maxjacobson**
Follow-up to #46, inspired by @ABaldwinHunter's comment https://github.com/codeclimate/codeclimate-fixme/pull/46#issuecomment-268617885

### CODE REVIEW

File: lib/fix-me.js [hunk 1]
```diff
@@ -35,6 +36,15 @@ FixMe.prototype.run = function(engineConfig) {
   this.find(paths, strings);
 };
 
+var isItsOwnConfig = function(path, lineNumber) {
```
Comment (ABaldwinHunter) [inline lib/fix-me.js]: wdyt about a different name for this function? 

I find the [test](https://github.com/codeclimate/codeclimate-fixme/pull/47/files#diff-353ae98a4e441f538fae0a4f9df49bccR63) description really informative.

WDYT about something like two functions:

`isItsOwnConfig` && `isNotComment` or something?

Comment (ABaldwinHunter) [inline lib/fix-me.js]: up to you though

**Comment by ABaldwinHunter** [comment]
One idea, up to you.

LGTM

File: lib/fix-me.js (hunk 1 above)
Comment (maxjacobson) [inline lib/fix-me.js]: Yea I like it - I named slightly differently, but on board with the idea

**Pull request merged**